internal/compiler/fileaccess.rs

```diff
@@ -29,13 +29,70 @@ pub fn styles() -> Vec<&'static str> {
 pub fn load_file(path: &std::path::Path) -> Option<VirtualFile> {
     match path.strip_prefix("builtin:/") {
         Ok(builtin_path) => builtin_library::load_builtin_file(builtin_path),
-        Err(_) => path.exists().then(|| VirtualFile {
-            canon_path: crate::pathutils::clean_path(path),
-            builtin_contents: None,
+        Err(_) => path.exists().then(|| {
+            let path =
+                crate::pathutils::join(&std::env::current_dir().ok().unwrap_or_default(), path)
+                    .unwrap_or_else(|| path.to_path_buf());
+            VirtualFile { canon_path: crate::pathutils::clean_path(&path), builtin_contents: None }
         }),
     }
 }
 
+#[cfg(test)]
+mod test {
+    pub struct ChDir {
+        current_dir: std::path::PathBuf,
+    }
+
+    impl ChDir {
+        pub fn new(path: &std::path::Path) -> Self {
+            let current_dir = std::env::current_dir().unwrap();
+            std::env::set_current_dir(path).unwrap();
```

Comment:
wait, i think this is a really bad idea to call in a test as tests are run in different thread and if several test do that this will not go well.